sceptre/context.py

```diff
@@ -1,7 +1,44 @@
+# -*- coding: utf-8 -*-
+
+"""
+sceptre.context
+
+This module implements the SceptreContext class which holds details about the
+paths used in a Sceptre project.
+"""
+
 from os import path
 
 
 class SceptreContext(object):
+    """
+    SceptreContext is a place that holds data that is relevant to the
+    project, including references to the project paths such as the path to your
+    Sceptre project, templates path, config path, and the default names for
+    your configuration files.
+
+    :param project_path: Absolute path to the base sceptre project folder
+    :param_type: str
```

Comment:
The `:param_type: <type>` annotation isn't consistent with a lot of other code which is using `:type <param>: <type>`.